lite: tiny typo-like update to ResizeBilinear opt op

zhenhuaw-me opened the pull request.

rthadur commented:
@jackwish Thanks for the PR , Here we can process only merge comit to tensorflow:master , please update it .

zhenhuaw-me committed:
lite: remove memset in resize bilinear opt op

memset() is not necessarily needed here since every element
in output_data memory will be updated.

zhenhuaw-me committed:
lite: perform std::floor before cast float to int

This won't change the resulted value, but to have same
code style in the context and reference_ops.

jackwish commented:
oops, pinged too many people, sorry... will open another.

jackwish commented:
> @jackwish Thanks for the PR , Here we can process only merge comit to tensorflow:master , please update it .

@rthadur check #24514 instead please.
